examples/src/main/scala/org/jtech/drone/ws/AsciiServiceActor.scala

```diff
@@ -24,11 +24,13 @@ class AsciiServiceActor extends Actor {
       println(s"Unhandled message $err")
   }
 
-  def broadcast(w: WsMessage): Unit =
-    if (lastSend < System.currentTimeMillis()) {
-      lastSend += 200
+  def broadcast(w: WsMessage): Unit = {
+    val currentTime = System.currentTimeMillis
+    if (currentTime - lastTime > 32) {
```

Comment:
Then I would take this logic out of here and put it in the feed generator. (`...feed.Main`). Can you please rename `...feed.Main` ⇒  `...feed.AsciiImageProducer` or something that you find suitable. 

My point is that this logic should be part of the `Feed` `Source`. If we want to throttle, we should do there instead of in server.